browser/js/app.js

```diff
@@ -6,6 +6,12 @@ app.config(function ($urlRouterProvider, $locationProvider) {
     $locationProvider.html5Mode(true);
     // If we go to a URL that ui-router doesn't have registered, go to the "/" url.
     $urlRouterProvider.otherwise('/');
+
+    
+    $urlRouterProvider.when('/auth/:provider', function () {
+        window.location.reload();
```

Comment:
I can't seem to make it work without the reload. Should we leave it in for now and merge this into master?